app/assets/stylesheets/4-molecules/_floors.scss

```diff
@@ -13,8 +13,7 @@
 		justify-content: center;
 		width: 100%;
 		overflow: hidden;
-		background-color: #D9F1F1;
```

Comment:
I removed the background because 2 things:
1. I feel with white background is more consistent with other views
2. I feel like green is not matching with it. (I'm not an expert on colors.)

![image](https://user-images.githubusercontent.com/7331511/168155115-425c6626-867e-43dd-b817-6f0d3dbd9f55.png)

![image](https://user-images.githubusercontent.com/7331511/168155181-57219ba9-79c7-4043-8929-cdb3ddca160c.png)